pkg/acsengine/testdata/disks-managed/swarm-vmss_expected.json

```diff
@@ -507,6 +507,13 @@
         "description": "SSH public key used for auth to all Linux machines.  Not Required.  If not set, you must provide a password key."
       },
       "type": "string"
+    },
+    "targetEnvironment": {
```

Comment:
same as my other comment: these should be AzureCloud, AzureChinaCloud, AzureUSGovernment, and AzureGermanCloud